helmcharts/kitchen/install.sh

```diff
@@ -97,6 +97,8 @@ additional)
     "azure")
         cp -rf ../services/azure-exporter additional/charts/
         ;;
+    *)
+        cp -rf ../services/minio additional/charts/
```

Comment:
priority needs change, ideally should be included when we install kong as other services will require it